execution/execution.go

```diff
@@ -70,3 +71,10 @@ func (execution *Execution) Fail(err error) error {
 	execution.Status = Status_Failed
 	return nil
 }
+
+// GetDuration returns the duration of the execution in seconds
+func (execution *Execution) GetDuration() int64 {
+	delta := execution.Stop - execution.Start
+	deltaInSecond := float64(delta) / 1e9
+	return int64(math.Ceil(deltaInSecond))
```

Comment:
For now the calculation is in second and not per 100ms in order to make the calculation simpler. We can change it in 100ms if it's critical but I don't think it is right now (at least in this PR)